conf_irods/partial_build.sh

```diff
@@ -7,52 +7,84 @@ echo "echo rods | iinit" | $RUN_ON_USER
 
```

Comment:
eventually will this script go away? was trying to reduce it's size not grow it. we can do that in a separate PR if it's too much work

Reply:
We will remove this script on other PR, this PR's purpose is only make life simpler for new developer